.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I would prefer if we wrap all around the DEV scope, inside dev folder.
Otherwise it will be mixed with TEST scope, inside test folder.

Reply:
probably I can remove it, IIUC this part was not working